src/main/java/com/hubspot/jinjava/tree/parse/Token.java

```diff
@@ -93,22 +88,33 @@ public String toString() {
 
   public abstract int getType();
 
-  static Token newToken(int tokenKind, String image, int lineNumber, int startPosition) {
-    switch (tokenKind) {
-      case TOKEN_FIXED:
-        return new TextToken(image, lineNumber, startPosition);
-      case TOKEN_NOTE:
-        return new NoteToken(image, lineNumber, startPosition);
-      case TOKEN_EXPR_START:
-        return new ExpressionToken(image, lineNumber, startPosition);
-      case TOKEN_TAG:
-        return new TagToken(image, lineNumber, startPosition);
-      default:
-        throw new UnexpectedTokenException(
-          String.valueOf((char) tokenKind),
-          lineNumber,
-          startPosition
-        );
-    }
+  static Token newToken(
+    int tokenKind,
+    TokenScannerSymbols symbols,
+    String image,
+    int lineNumber,
+    int startPosition
+  ) {
+    if (tokenKind == symbols.TOKEN_FIXED()) return new TextToken(
```

Comment:
please always wrap bodies of if statements in `{ }`

Reply:
done.